3p/vendors/exco.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright 2021 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// src/polyfills.js must be the first import.
+import '../polyfills';
```

Comment:
we changed these import paths while this was pending, sorry. The new structure should be something like 
```js
import '#3p/polyfills';

import {draw3p, init} from '#3p/integration-lib';
import {register} from '#3p/3p';

import {exco} from '#ads/vendors/exco';
```

Reply:
Done